components/Common/LanguageSelector/__tests__/index.test.tsx

```diff
@@ -0,0 +1,34 @@
+import { fireEvent, render, screen } from '@testing-library/react';
+import LanguageSelector from '..';
+
+jest.mock('../../../../hooks/useLocale', () => ({
```

Comment:
Interesting approach to mock the `useLocale` hook. I like this! (Most of other contributors went the path of just injecting the `IntlProvider` but with no arguments)